Source/Core/Core/NetPlayServer.cpp

```diff
@@ -414,10 +415,10 @@ void NetPlayServer::ThreadFunc()
   INFO_LOG_FMT(NETPLAY, "NetPlayServer shutting down.");
 
   // close listening socket and client sockets
-  for (auto& player_entry : m_players)
+  for (auto const& entry : std::views::values(m_players))
```

Comment:
AFAICT, the codebase doesn't seem to use the `auto const&` syntax much, so `const auto&` should be preferred for consistency.